nebula-css/_settings.scss

```diff
@@ -6,6 +6,7 @@ $nb-spacing-unit: 1rem !default;
 $nb-spacing-unit-half: ($nb-spacing-unit / 2) !default;
 $nb-spacing-unit-double: ($nb-spacing-unit * 2) !default;
 
+$nb-breakpoint-class: '\\@' !default;
```

Comment:
I don't think so, since the first `\` is escaping the "actual" `\` from the string (so the actual string value is `\@`), otherwise we'd try to escape the `@` itself which is not what we want.